operator/pkg/controllers/master/authenticatorconfig.go

```diff
@@ -108,6 +108,20 @@ func (c *Controller) reconcileAuthenticatorDaemonSet(ctx context.Context, contro
 								Name:      "kubeconfig",
 								MountPath: "/var/aws-iam-authenticator/kubeconfig/",
 							}},
+							LivenessProbe: &v1.Probe{
+								Handler: v1.Handler{
+									HTTPGet: &v1.HTTPGetAction{
+										Host:   "127.0.0.1",
+										Scheme: v1.URISchemeHTTP,
+										Path:   "/healthz",
```

Comment:
Does this component separate readiness/liveness?

Reply:
For authenticator we don't connect through a service, so readiness might not be as useful.

Reply:
Good point, will add a readiness probes to apiserver and etcd pods since traffic to these pods is sent through services.